packages/visualizations-react/.eslintrc.js

```diff
@@ -1,8 +1,107 @@
+            {
+                code: 120,
+                tabWidth: 4,
+                ignoreStrings: true,
+                ignoreTemplateLiterals: true,
+                ignoreComments: true,
+            },
+        ],
+        camelcase: 0,
+        'no-plusplus': 0,
+        // disallow reassignment of function parameters
+        // disallow parameter object manipulation except for specific exclusions
+        'no-param-reassign': [
+            'error',
+            {
+                props: true,
+                ignorePropertyModificationsFor: [
+                    'acc', // for reduce accumulators
+                    'e', // for e.returnvalue
+                ],
+            },
+        ],
+        'object-curly-newline': [
+            'error',
+            {
+                ObjectExpression: {
+                    minProperties: 6,
+                    multiline: true,
+                    consistent: true,
+                },
+                ObjectPattern: {
+                    minProperties: 6,
+                    multiline: true,
+                    consistent: true,
+                },
+            },
+        ],
+        'import/no-named-as-default': 'off',
+        'import/no-extraneous-dependencies': [
+            'error',
+            { devDependencies: ['./test/**', '/stories/**'] },
+        ],
+        'no-underscore-dangle': 0,
+        'react/jsx-one-expression-per-line': 0,
+        'arrow-parens': [2, 'as-needed', { requireForBlockBody: true }],
+        'react/jsx-props-no-spreading': 0,
+        'jsx-a11y/label-has-for': 0,
+        'jsx-a11y/label-has-associated-control': ['error', {
+            labelComponents: ['label'],
+            controlComponents: ['input', 'CheckboxInput', 'RadioInput', 'SelectInput', 'TextInput', 'SwitchInput'],
+            depth: 25,
+        }],
+        'react-hooks/rules-of-hooks': 'error',
+        'react-hooks/exhaustive-deps': 'warn',
+        'object-curly-newline': ['error', {
+            ObjectExpression: { minProperties: 6, multiline: true, consistent: true },
+            ObjectPattern: { minProperties: 6, multiline: true, consistent: true },
+        }],
+        'react/jsx-filename-extension': [2, { extensions: ['.jsx', '.tsx'] }],
+        'react/react-in-jsx-scope': 'off',
+        'import/no-named-as-default': 'off',
+        // FIXME: With the introduction of TS, these rules cause linter errors (many of them quite justified). For the
+        // ... moment, they have been disabled but in the future we could re-enable them and fix the different errors.
+        '@typescript-eslint/naming-convention': 'off',
+        '@typescript-eslint/default-param-last': 'off',
+        '@typescript-eslint/no-var-requires': 'off',
+        // avoid the usage of test.only
+        'no-only-tests/no-only-tests': 'error',
+        "react/function-component-definition": [0, { namedComponents: ["function-expression", "arrow-function"], unnamedComponents: "arrow-function" }],
```

Comment:
Why not leave the default value for this rule? Personal preference? (If yes, it's fine with me, I'm asking mostly out of curiosity ^^). The definition of this rule in the platform is:
```
"react/function-component-definition": [0, { namedComponents: 'function-expression', unnamedComponents: 'function-expression'}],
```

Reply:
I think there was a slight looping problem with Prettier here, but yes personal preference also, I really don't like function declaration for components (it made stories really strange) and I really prefer arrow for anything unamed/inlined. But that's pretty open to discussion…

Reply:
As I don't have any opinion on the subject, and in the spirit of running ahead, I suggest we don't touch anything for now and go with this 😅